v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,280 @@
+
+- Anyone who has the public key can use it to encrypt a message and send it to you, knowing that you alone can decrypt it with the private key.
+- Anyone who receives a message that can be decrypted correctly using the public key knows it *must* have been encrypted with the private key, meaning it must have come from you.
+
+TLS therefore provides two critical security features:
+
+- **Encryption**: it prevents message from being eavesdropped or tampered with.
+- **Authentication**: it allows one or both parties to prove their identity that cannot be impersonated (or 'spoofed' in cyberspeak).
+
+### Mutual vs one-sided TLS authentication
+
+TLS connections can be either *one-sided*, meaning that only one party must prove its identity before an encrypted session is established, or *two-sided* (or *mutual*), meaning that both must prove their identities.
+
+In mutually authenticated TLS connections, each party must have a key pair issued by a jointly trusted CA. This is a good system for internal components within a distributed system, and generally works well for long-lived clients.
+
+In one-sided TLS authentication, only the server must have a key pair; the user may be able to access information without any authentication. This works well when:
+
+- The client needs no authentication, for example, for a public read-only website or API.
+- When the user must authenticate with another mechanism, such as a username/password combination or Single Sign-On (SSO).
+
+### Symmetric and asymmetric encryption in TLS
+
+Encryption using a key pair is "asymmetric", in that a different key is used to encrypt and decrypt. Encryption is called "symmetric" where the same key is used to encrypt and decrypt&mdash;as the metal key in a mechanical lock-and-key system is used to both lock and unlock.
+
+Symmetric encryption is more efficient than asymmetric encryiption, but it requires all parties to have the single key shared key prior to the exchange.
+
+With asymmetric encryption, by contrast, each party can share their public key, allowing anyone to send them messages that only they can decrypt with their private key. In the context of internet communication
+
+In the TLS protocol, asymmetric encryption using the public/private key pairs is used to securely exchange random data from that is used to create shared **session keys**, which are then used by both parties to (symmetrically) encrypt and decrypt data for the remainder of the session. Since a session key is created with information shared using asymmetric encryption, and is used only for the duration of a session, TLS combines the best of both asymmetric encryption (the ability to establish an encrypted communication channel without having to have first securely shared a key) and symmetric encryption (computational efficiency).
```

Comment:
I'm having a hard time parsing this paragraph. I think maybe some things got jumbled during rewriting.

Reply:
hope that reads better.